src/sys/ptrace.rs

```diff
@@ -76,6 +77,7 @@ pub fn ptrace(request: ptrace::PtraceRequest, pid: pid_t, addr: *mut c_void, dat
 
     match request {
         PTRACE_PEEKTEXT | PTRACE_PEEKDATA | PTRACE_PEEKUSER => ptrace_peek(request, pid, addr, data),
+        PTRACE_GETEVENTMSG => ptrace_get_data::<c_long>(request, pid),
```

Comment:
It was originally but I've of the review comments was that it seemed an unnecessary function. Unless I misunderstood?